barrels/LogMonkey/Logger.mc

```diff
@@ -0,0 +1,172 @@
+using Toybox.Time.Gregorian as Greg;
+using Toybox.Lang as Lang;
+
+module LogMonkey {
+
+    //! The Logger class provides public APIs to log various types of
+    //! messages. Each Logger class is associated with a log level which
+    //! is set within the Logger.initialize() function.
+    class Logger {
+
+        private var mLogLevel, mLogStream;
```

Comment:
These private static members are not necessary in release. They could easily be annotated out for additional savings.

Reply:
Perhaps it makes sense to have two versions of this class, one annotated with :release that just declares empty versions of the public methods and another that is annotated with :debug that has all of the necessary implementation details.

Reply:
I like the idea of separating this by class. I'm going to take that approach.